pkgs/tools/security/cryptomator/default.nix

```diff
@@ -6,13 +6,13 @@
 
 let
   pname = "cryptomator";
-  version = "1.6.14";
+  version = "1.8.0";
 
   src = fetchFromGitHub {
     owner = "cryptomator";
     repo = "cryptomator";
     rev = version;
-    sha256 = "sha256-ArOYL3xj2HiXXu1Bymd5mciMsmikCDvxr5M3LMqZgYA=";
+    sha256 = "sha256-2nCaSL7OlS9f+PZPh0YiMvnjOaAqlQimkKWDSjSP+bQ=";
```

Comment:
Still wrong according to ofborg.

Reply:
And ofborg now reports this hash. This is so strange!

Reply:
Agree, can we request orgborg to review one more time just in case it changes?

Reply:
I get `sha256-2nCaSL7OlS9f+PZPh0YiMvnjOaAqlQimkKWDSjSP+bQ=` as a hash when running on x86-64-linux.

Reply:
since is one of my first  time contributing and correcting, what is the best aproach to handle this?

Reply:
Let's drop the last commit then...

Reply:
@discite I suggest to [rebase](https://git-scm.com/book/en/v2/Git-Branching-Rebasing) and drop all but the first commit.

Reply:
@discite You most probably also need to change the hash of the dependencies. See line 40.

Reply:
#230757